src/components/Header/index.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react'
+import { Provider } from 'react-redux';
+import store from '../../state/store';
+import Header from './index'
+
+test("render header bar", async () => {
+  const { container } = render(<Provider store={store}><Header /></Provider>);
+  expect(container).toBeInTheDocument();
```

Comment:
It should be more detailed.

Such as:
- Which text is in the document?
- Which component is in the document?
- ...